### PR TITLE
HOPSWORKS-233 and HOPSWORKS-242

### PR DESCRIPTION
**Pull request by jimdowling**
Tensorboard
Jupyter Python Kernel - one per ProjectUser.

### CODE REVIEW

File: hopsworks-web/yo/app/scripts/controllers/homeCtrl.js [hunk 1]
```diff
@@ -287,7 +289,7 @@ angular.module('hopsWorksApp')
                           growl.warning("some problem", {title: 'Error', ttl: 10000});
                         }
                       },
-                      function (error) {
+                      function (error) {showTourTips
```
Comment (tkakantousis) [inline hopsworks-web/yo/app/scripts/controllers/homeCtrl.js]: @jimdowling why is showTourTips here?

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java [hunk 2]
```diff
@@ -220,6 +222,8 @@ public Project createProject(ProjectDTO projectDTO, Users owner,
         }
       }
     }
+    LOGGER.log(Level.INFO, "PROJECT CREATION TIME. Step 1: " + (System.currentTimeMillis() - startTime));
```
Comment (tkakantousis) [inline hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java]: @jimdowling These print outs must be set to debug level.

File: hopsworks-web/yo/app/scripts/controllers/homeCtrl.js (hunk 1 above)
Comment (jimdowling) [reply hopsworks-web/yo/app/scripts/controllers/homeCtrl.js]: looks like a typo. thanks.

**Comment by jimdowling** [comment]
@tkakantousis  lgtm (lets get this merged)

**Pull request merged**